src/app/http/http.service.ts

```diff
@@ -1,7 +1,25 @@
+import { YtResponse } from '../interfaces/youtubeResponse';
+import { HttpClient } from '@angular/common/http';
 import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { Movie } from '../interfaces/movie';
```

Comment:
jeśli zrobiłeś poprawnie plik interfaces/index.ts, to powinieneś być w stanie zaimportować `import { Movie, ytResponse } from '../interfaces';